xs/src/libslic3r/PrintConfig.cpp

```diff
@@ -775,6 +775,15 @@ PrintConfigDef::PrintConfigDef()
     def->max = 100;
     def->default_value = new ConfigOptionInt(35);
 
+    def = this->add("min_shell_thickness", coFloat);
+    def->label = "Minimum shell thickness";
+    def->category = "Layers and Perimeters";
+    def->sidetext = "mm";
+    def->tooltip = "Minimum shell thickness in mm";
```

Comment:
This needs to be more clear. Minimum shell thickness is an alternative way for configuring perimeters and solid shells. If it's set to a non-zero value, they will be automatically increased until the configured value is reached.